geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/ScaldingDelimitedIngestJob.scala

```diff
@@ -127,7 +128,7 @@ class ScaldingDelimitedIngestJob(args: Args) extends Job(args) with Logging {
   // Check to see if this an actual ingest job or just a test.
   if (!isTestRun) {
     new MultipleUsefulTextLineFiles(pathList: _*).using(new Resources)
-      .foreach('line) { (cres: Resources, line: String) => lineNumber += 1; ingestLine(cres.fw, line) }
+      .foreach('line) { (cres: Resources, line: String) => lineNumber += 1; ingestLine(cres.fw, line, skipHeader) }
```

Comment:
Maybe you could do a if statement to use a ```.drop(1)``` then the ```.foreach``` if the skipheader is there